templates/base.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>{% block title %}Taxi Service{% endblock %}</title>
```

Comment:
The `title` block is correctly implemented with the title 'Taxi Service', fulfilling the task requirement to create a block `title` with `Taxi Service` inside.